CHANGELOG.md

```diff
@@ -1,33 +1,38 @@
-## 1.12.0-rc2 / 2021-02-07
+## 1.12.0 / 2021-02-11
 
-* [BUGFIX] fixed mixed case keys for key-value, all keys are lower case now (#1778)
+WARNING: The self updater does not support updating from 1.11.0 to 1.12.0. Our
+release infrastructure does not support the key type used in 1.11.0.
+
+NOTE: This release drops the integrations that were moved to their own repos.
```

Comment:
I would name them explicitly. I'm not sure a Summon user would understand what it means for example.